### DISCUSSION TITLE
我也可以預測地震嗎?   或是讓AI預測地震呢?

### DISCUSSION THREAD

**Discussion by HuanYuuuuuu** [1101-GS4719-C/Python-Community-GS4719-C#512] [General]
如題，普通人也可以像印度神童那樣預測災難嗎?
但其實在台灣 根據氣象法規定 任何人都不能發布警報或災害性天氣中的豪雨及颱風警報
如擅自發布 氣象局可依情節輕重處罰20萬~100萬
但如果你沒有根據氣象相關研究或是相關領域認可的方式預測 而是剛好做夢夢到 或是媽祖托夢給你 這樣子的個人感覺會有地震發生的話 是沒有違法的。所以如果哪天你覺得今天可能會發生地震! 你可以放心的在IG FB上發文 沒有人可以罰你錢

地震其實是一種微觀前兆很多的災害，其中涉及了地殼運動、磁場、重力場、地下水等等。透過專業儀器來隨時偵測這些數據。AI則需要透過機器學習以及演算法來找出哪些是可用數據。就可以有機會可以大概在地震發生前20s預測出地震何時、何地發生。
(只是精準度就不一定了.... 目前似乎只有在實驗室可以成功地預測出下一次地震的時間)

感覺AI結合地震預測是一個還不錯的發展方向，畢竟台灣處於地震帶上，如果我們有方法可以預測到天災的到來，也許就可以保護到我們所愛的人。

**Comment by Sazmafuji** [comment]
如果能夠預測地震，像昨天晚上的地震，我們也不需要特地起來擔心，說到預測地震，我聽過我們是能夠依照之前的地震大致判斷地震會在幾年內發生，但沒辦法知道會在哪發生，如果再加上能夠預測位置，想必會有更進一步的發展

**Comment by Lai27330** [comment]
地震發生前有太多不同的變數了，感覺即使涵蓋所有可能去預測的話，最好可能是像下雨的機率一樣，時間再拉長的話準確率也會快速降低，但之前有看到學校的地科學系有新的進步，希望未來有一天能發展為成熟的技術
http://esrpc.ncu.edu.tw/public/tw/focus/focus_detail/975

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望技術可以更加成熟。

**Comment by hapiwangy** [comment]
近期地震發生的十分頻繁，若能有相關的科技能夠成熟，相信我們也能夠有更安全的生活

**Comment by HaoFish123** [comment]
感謝分享，地震真的很可怕。

**Comment by PoooYen** [comment]
希望日後能提早預測讓我們好避難

**Comment by xkllkx** [comment]
希望這項技術能夠早日成熟，台灣十分需要這項技術

**Comment by yenwei12138** [comment]
現在台灣已經有不少學者在研究相關領域了。目前的耐震預測大多結合深度學習對建築物過去的受震反映來分析地震模式並用於結構物耐震預測及分析。
https://www.digitimes.com.tw/iot/article.asp?cat=158&id=0000569386_kbi4dmwp76dfz0720mg2a&fb_comment_id=2552098751521136_4177330518997943

**Comment by Lynn12378** [comment]
昨天剛好參加一場有關地震的大一周會，講師有提到目前來說預測地震太過困難，地震化石的樣本很容易被風化，可以參考的樣本數不夠支撐預測地震成因的假設，加上地震變因受到地形的影響所以太複雜，短期內預測地震對於人類還是一大難題。目前只能在地震發生的第一瞬間將資訊傳至可能發生重大災情的地區。

**Comment by Yuhsin000** [comment]
20秒感覺可以大大增加民眾逃生的時間耶(畢竟還要扣除通報及發送訊息的時間)

**Comment by ghost** [comment]
雖然先在地震大多都是抱持著真的震很大大概也跑不了的心態，但如果真的能有充裕時間預知地震，我想我的求生慾應該會高不少

**Comment by Ben6uo** [comment]
感謝分享，希望可以更快普及,地震真的很可怕

**Comment by cordydeng** [comment]
這個技術如果能更成熟 , 想必一定可以造福全部人 , 不過我們也得在平常時做好準備 , 才不會在地震發生時太過恐慌

**Comment by sys511613** [comment]
現在已經能透過網路，在地震發生的當下透過快速的網路告知大家，對於其他地區的人可以在地震到來前幾秒就做好準備
希望這個技術能早日成熟，對於未來面對地震的準備和預防都會更有幫助

**Comment by jcpan821** [comment]
希望以後預測的時間能在更長，不然根本來不及跑

**Comment by alexthebigpp** [comment]
地震感覺還是有很多細微的徵兆的，只是人類感覺不到，但是透過AI可以把這些小細節抓出來提前預警

**Comment by minniewei** [comment]
地震發生真的很可怕，但假如有這種技術感覺會減輕恐慌感

**Comment by Jimmy911218** [comment]
如果這項技術更成熟了將會避免更多的921
